StPetersburg/IVEf17.xml

```diff
@@ -0,0 +1,301 @@
+                      </desc>
+                     </decoNote>
+
+                     <decoNote type="frame" xml:id="d2">
+                        <locus target="#78r"/>
+                        <desc>Coloured ornamental frame.</desc>
+                     </decoNote>
+
+                     <decoNote type="frame" xml:id="d3">
+                        <locus target="#112v"/>
+                        <desc>Coloured ornamental frame.</desc>
+                     </decoNote>
+
+                     <decoNote type="frame" xml:id="d4">
+                        <locus target="#129r"/>
+                        <desc>Coloured ornamental frame.</desc>
+                     </decoNote>
+
+                  </decoDesc>
+
+                  <additions>
+                     <list>
+                        <item xml:id="a1">
+                           <locus from="2r" to="3v"/>
+                           <desc type="GuestText">
+                             <title type="complete" ref="LIT2839RepCh123"/>
+                           </desc>
+                             <q xml:lang="gez">ሰላም፡ ለዝክረ፡ ስምከ፡ ስመ፡ መሐላ፡ ዘኢይሔሱ
+                             </q>
+                              </item>
+
+                              <item xml:id="a2">
+                                <locus from="4r" to="5v"/>
+                                 <desc type="GuestText">
+                                   <title type="complete" ref="LIT2444Temher"/>
+                           </desc>
+                                 <q xml:lang="gez">በእንተ᎓ ትምህርት᎓ ህቡአት᎓
+                                   </q>
+                                    </item>
+
+                                    <item xml:id="a3">
+                                      <locus target="#139v"/>
+                                      <desc type="GuestText">
+                                        <title type="complete" ref="LIT3050RepCh330"/>
+                                      </desc>
+                                          <q xml:lang="gez">ኢየሱስ᎓ ክርስቶስ᎓ ወልደ᎓ እግዚአብሔር᎓ በዘባነ᎓ ባሕር᎓ ከመ᎓ ትትለሀይ᎓ ሐመር᎓
+                                              መንክር᎓ ተላህይ᎓ ለዘለከ᎓ ፍቅር፨
+                                         </q>
+                                         <note>The hymn is composed of 15 strophes. </note>
+                                          </item>
+
+                              <item xml:id="a4">
+                                <locus target="#140"/>
+                                <desc type="GuestText">
+                                  <title type="incomplete" ref="LIT2866RepCh147"/>
+                           </desc>
+                                <q xml:lang="gez">ሰላም᎓ ለዝክረ᎓ ስምከ᎓ ለሰሌዳ᎓ ሞገስ᎓ መጽሐፍ᎓
+                                  </q>
+                              </item>
+
+
+                     </list>
+                  </additions>
+
+                  <bindingDesc>
+                     <binding contemporary="false" xml:id="binding">
```

Comment:
just to be sure: The catalogue mentions that this binding is not the original one?

Reply:
ok